www/content/tutorials/cli.md

```diff
@@ -7,3 +7,98 @@ weight = 4
 +++
 
 {{< wip >}}
+
+# tricorder
+This ist the main Command. 
+This Command specifies, which hosts are affected by the subcommand.
```

Comment:
"Command", this is a noun, so no capitalization.

The description is unclear, I'd go with the description present in the API documentation: https://docs.rs/tricorder/latest/tricorder/cli/index.html